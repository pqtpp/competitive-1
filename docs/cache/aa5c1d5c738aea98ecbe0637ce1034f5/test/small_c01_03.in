193 273
152 119 0
152 192 1
119 60 0
152 182 0
152 50 1
119 124 0
50 14 0
60 9 0
152 18 1
192 132 0
124 162 0
14 190 1
192 108 0
14 76 0
76 130 1
14 173 1
18 98 1
119 40 0
18 110 1
9 107 0
60 90 0
50 82 0
152 0 1
162 8 1
50 44 1
152 104 0
162 31 1
132 180 0
173 156 0
82 169 1
9 64 0
90 120 1
169 46 1
14 20 0
50 191 0
173 145 0
107 41 0
98 74 1
173 80 0
90 116 0
173 155 0
108 166 1
182 63 0
108 117 0
162 29 0
110 137 0
40 126 1
98 78 0
116 85 1
78 62 1
80 186 0
74 185 1
117 139 0
64 102 1
20 42 1
60 121 1
104 150 1
60 38 1
44 159 1
0 70 1
50 2 0
150 148 1
148 168 1
29 94 0
98 118 1
31 59 0
155 161 1
29 189 1
137 23 0
124 147 1
180 153 0
121 11 0
147 75 1
98 176 1
41 25 1
150 24 1
46 57 0
38 5 1
85 66 0
130 30 1
75 83 0
63 58 0
139 163 1
44 164 1
38 4 1
25 19 0
116 49 0
137 26 0
62 27 0
60 65 1
94 99 0
75 92 0
99 154 1
19 32 0
2 71 0
62 112 1
163 36 0
2 113 0
60 183 1
189 77 0
78 143 1
74 51 1
186 86 1
83 95 1
186 15 1
15 91 0
19 172 1
41 175 1
183 133 0
70 115 0
27 17 1
25 97 0
112 10 1
159 67 1
27 181 0
116 142 0
163 179 1
115 167 1
60 6 0
9 141 1
95 84 0
17 149 0
27 138 0
70 184 1
155 128 1
20 157 0
142 89 0
124 146 0
150 37 1
145 111 1
2 13 1
154 53 0
172 87 0
29 151 0
126 114 1
156 177 1
84 68 1
74 174 1
114 136 1
46 43 1
141 69 1
31 28 0
185 47 1
112 48 0
153 109 0
78 3 0
14 52 1
110 79 0
175 88 0
128 131 1
75 103 0
92 33 1
153 35 0
67 16 1
168 158 0
108 140 0
60 45 1
184 55 1
109 22 0
141 101 0
104 178 1
110 144 1
107 122 0
158 96 0
182 123 0
63 125 0
65 7 1
189 1 0
176 127 0
20 56 1
47 187 1
86 165 1
142 134 0
33 72 0
176 106 1
168 73 0
177 171 0
22 129 0
25 81 0
4 105 0
59 61 0
42 21 0
88 160 1
133 54 0
71 12 0
177 188 0
47 34 0
45 135 1
96 100 1
154 39 0
109 170 1
182 93 1
144 55 0
174 49 0
64 155 0
28 107 1
22 32 0
117 167 0
82 67 1
125 33 1
12 145 0
89 20 0
162 73 1
45 157 0
174 19 1
118 157 0
99 1 1
177 39 0
161 19 1
50 18 0
130 41 0
48 186 1
172 148 1
134 22 0
93 27 1
116 153 0
134 36 1
175 79 0
101 1 1
152 23 0
11 177 1
85 109 0
29 60 1
24 110 0
87 138 0
141 80 0
39 76 1
4 165 0
100 3 1
173 145 1
40 144 1
118 179 0
13 55 1
151 57 1
162 186 1
158 64 0
42 43 0
189 72 1
21 77 0
77 21 1
30 15 0
37 109 1
49 73 1
129 148 0
34 157 1
63 162 0
31 102 0
59 144 1
87 50 1
121 186 1
189 162 0
67 102 1
98 95 1
57 189 0
63 117 1
88 130 1
111 108 0
175 58 0
88 183 1
148 59 0
146 136 1
74 71 0
79 150 1
159 34 1
131 29 0
131 128 0
0 5 0
77 34 0
138 82 1
91 64 1
3 36 1
32 23 0
152 118 1
