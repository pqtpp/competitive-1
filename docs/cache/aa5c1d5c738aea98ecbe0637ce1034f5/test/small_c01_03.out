56
266 22 0 2 3 5 7 10 17 19 20 25 30 36 39 42 44 63 81 86 90 115 118 126 127 133 162 164 165 172 193 194 40 28 35 38 50 200 184 94 60 6 13 21 33 34 97 125 203 205 209 210 213 158 144 70 27 9 12 43 52 152 155 177 196 93 85 111 178 197 219 68 45 87 147 217 148 221 78 223 225 159 231 235 154 163 175 243 250 99 167 238 181 236 253 76 256 259 65 141 180 246 261 264 265 267 186 11 16 47 145 23 24 29 31 41 46 49 88 114 122 224 132 51 55 71 56 57 58 59 109 66 69 72 80 91 150 73 168 79 240 105 82 96 83 84 179 227 89 92 131 189 207 176 185 95 143 98 108 183 100 101 102 103 120 110 121 112 113 123 128 130 134 136 138 140 146 151 173 153 157 192 160 166 169 170 174 182 187 188 190 191
